fusion.md

```diff
@@ -1176,7 +1176,8 @@ to use `STATE` with `SIMPLE` or `ARRAY`.  You should add this flag to any Da
 definition who's data will get set during a call to `World::calc`, as described
 in [Centrally Managed State Variables](#centrally-managed-state-variables).
 
-
+### Add your new class to `gcam_data_containers.h`
```

Comment:
Well.. I guess we were going to have to merge these changes in gcam5.0 at some point.  Might as well come along with hector.